cdc/entry/schema_storage.go

```diff
@@ -160,12 +178,15 @@ func (s *schemaStorageImpl) HandleDDLJob(job *timodel.Job) error {
 	var snap *schema.Snapshot
 	if len(s.snaps) > 0 {
 		lastSnap := s.snaps[len(s.snaps)-1]
-		if job.BinlogInfo.FinishedTS <= lastSnap.CurrentTs() {
+		if job.BinlogInfo.FinishedTS <= lastSnap.CurrentTs() || job.BinlogInfo.SchemaVersion <= s.schemaVersion {
```

Comment:
When can `job.BinlogInfo.SchemaVersion <= s.schemaVersion` happen?

Reply:
when a duplicate job is received

Reply:
Please add some comments on it.
It's related to the frame switch of `job table` and `job queue`.